server/node/src/main/java/org/kaaproject/kaa/server/node/service/credentials/CredentialsServiceAdapter.java

```diff
@@ -19,20 +19,20 @@
 import java.util.Optional;
 
 import org.kaaproject.kaa.common.dto.credentials.CredentialsDto;
+import org.kaaproject.kaa.server.common.dao.exception.CredentialsServiceException;
 
 /**
- * A bridge between the {@link CredentialsService} and
- * {@link InternalCredentialsService} interfaces.
+ * A bridge between the actual interface and the internal Kaa implementation.
  *
  * @author Andrew Shvayka
  * @author Bohdan Khablenko
  *
  * @since v0.9.0
  */
-public final class InternalCredentialsServiceAdapter implements CredentialsService {
+public final class CredentialsServiceAdapter implements CredentialsService {
 
     private final String applicationId;
-    private final InternalCredentialsService credentialsService;
+    private final org.kaaproject.kaa.server.common.dao.CredentialsService credentialsService;
```

Comment:
This is not the best approach, but since it is not affecting any other places in code, we can probably leave it as is.